run-network/creator.py

```diff
@@ -0,0 +1,473 @@
+import re
+import subprocess
+import time
+from collections import Counter
+from dataclasses import dataclass, field
+from os import chdir, environ, chmod
+from os.path import dirname
+from pathlib import Path
+from typing import Any, Dict, ForwardRef, List, Optional, Type
+
+import click
+import requests
+import yaml
+
+
+@click.group()
+def cli():
+    """A simple CLI program"""
+    pass
+
+
+@cli.command("create-docker")
+@click.argument("config_path")
+def create_docker_cli(config_path: str):
+    """Creates a docker-compose file"""
+    config: Config = Config.from_file(config_path)
+    create_docker_compose(config)
+
+
+@cli.command("create-env")
+@click.argument("config_path")
+def create_env_cli(config_path: str):
+    """Creates .env for docker-compose"""
+    config: Config = Config.from_file(config_path)
+    create_env(config.environment, config.dest_env)
+
+
+@cli.command()
+@click.argument("config_path")
+def start(config_path):
+    """Start docker-compose in base_dir"""
+    config: Config = Config.from_file(config_path)
+    dest = Path(config.dest)
+    chdir(dest.parent)
+    environ["PWD"] = str(dest.parent)
+    command = ["docker-compose", "-f", f"{dest.name}", "up", "-d"]
+    subprocess.run(command)
+    wait_for_startup(f"http://localhost:{config.environment.ocrd_ps_port}")
+
+
+@cli.command()
+@click.argument("venv_bin_path")
+@click.argument("config_path")
+def create_clients(venv_bin_path: str, config_path: str):
+    """ Creates a script for every processor to call and ocrd-process for workflow runs
+
+    The processing server and the workers run in docker. To simplyfy the invocation a delegator for
+    every existing worker is created. These scripts are added to the venv's bin directory.
+    """
+    if not Path(venv_bin_path).exists():
+        exit(f"path to venv not found: {venv_bin_path}")
+    elif not Path(config_path).exists():
+        exit(f"path to config file not found: {config_path}")
+
+    config: Config = Config.from_file(config_path)
+    port = config.environment.ocrd_ps_port
+
+    for proc in config.processors:
+        content = re.sub(
+            r"PROCESSOR_NAME\s*=\s*\"[^\"]+\"",
+            f'PROCESSOR_NAME = "{proc.name}"',
+            DELEGATOR_PROCESSOR_TEMPLATE.lstrip(),
+            1
+        )
+        content = re.sub(
+            r"PROCESSING_SERVER_PORT\s*=\s*[0-9]+",
+            f"PROCESSING_SERVER_PORT = {port}",
+            content,
+            1
+        )
```

Comment:
So here it would simply become:
```suggestion
        content = DELEGATOR_PROCESSOR_TEMPLATE.format(processor_name=proc.name, ps_port=port)
```